### DISCUSSION TITLE
Builds & deployments are down

### DISCUSSION THREAD

**Discussion by tnsengimana** [vercel/community#1999] [Help]
### Summary

Vercel is failing to make external calls.

Sample build log (note: replaced our API URL wit `<endpoint>`; our API is working fine):
```
20:31:54.754 | message: 'getaddrinfo ENOTFOUND <endpoint>,
20:31:54.754 | status: 100
```

Sample runtime log:
```
Invoke Error 	{"errorType":"Error","errorMessage":"socket hang up","code":"ECONNRESET","stack":["Error: socket hang up","    at connResetException (node:internal/errors:718:14)","    at Socket.socketOnEnd (node:_http_client:520:23)","    at Socket.emit (node:events:525:35)","    at Socket.emit (node:domain:489:12)","    at endReadableNT (node:internal/streams/readable:1359:12)","    at process.processTicksAndRejections (node:internal/process/task_queues:82:21)"]}
```

Is the issue affecting us only?

**Comment by amyegan** [comment]
Hi @tnsengimana. I haven't seen any similar reports of build & deploy problems today, but I'll watch out for any that may come later. 

First, you should double check that your API-related environment variables are valid. If you already ruled out expired tokens or an external service outage, the next thing to try is a [minimal reproduction](https://vercel.com/support/articles/creating-a-minimal-reproducible-example) so we can get a better look at the problem.

**Comment by tnsengimana** [comment]
I found the root cause. Thank you for the comment!
